crypto/ec/ecx_meth.c

```diff
@@ -195,7 +195,7 @@ static int ecx_priv_decode(EVP_PKEY *pkey, const PKCS8_PRIV_KEY_INFO *p8)
     }
 
     rv = ecx_key_op(pkey, pkey->ameth->pkey_id, palg, p, plen, KEY_OP_PRIVATE);
-    ASN1_OCTET_STRING_free(oct);
+    ASN1_STRING_clear_free(oct);
```

Comment:
Uhm, I don't know the gritty details of the ASN1 macros very well.

Is it OK to use an `ASN1_STRING_foo()` function on a `ASN1_OCTECT_STRING` object?

Reply:
They're the same thing. From ossl_typ.h:

````
typedef struct asn1_string_st ASN1_OCTET_STRING;
...
typedef struct asn1_string_st ASN1_STRING;
````

Reply:
Thanks Matt!